ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -263,28 +300,45 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'R2S Delayed Partners/Spread Callout',
+      id: 'R2S Honey Beeline Initial',
       type: 'StartsUsing',
-      netRegex: { id: ['9184', '9185', '9B08', '9B09'], source: 'Honey B. Lovely', capture: false },
-      delaySeconds: (data) => {
-        // TODO: Review these delay timings
-        switch (data.partnersSpreadCounter) {
-          case 1:
-            return 14;
-          case 2:
-            return 11;
-          case 3:
-            return 37;
-          case 4:
-            return 62;
-          case 5:
-            return 55;
-        }
-        return 0;
+      netRegex: { id: ['9186', '9B0C'], source: 'Honey B. Lovely', capture: false },
+      response: Responses.goSides(), // default is alertText, no need to specify
+    },
```

Comment:
this block seems like a duplicate that doesn't need to be added.

Reply:
That, or remove the original report.

Reply:
> this block seems like a duplicate that doesn't need to be added.

> That, or remove the original report.

I'm not sure what you mean.  A duplicate of what?  What's the "original report" that you're referring to?